src/fu-tool.c

```diff
@@ -1673,15 +1673,17 @@ fu_util_get_remotes (FuUtilPrivate *priv, gchar **values, GError **error)
 	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
 		return FALSE;
 
+	/* list remotes */
 	remotes = fu_engine_get_remotes (priv->engine, error);
 	if (remotes == NULL)
 		return FALSE;
 	if (remotes->len == 0) {
-		/* TRANSLATORS: no repositories to download from */
-		g_print ("%s\n", _("No remotes available"));
-		return TRUE;
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_NOTHING_TO_DO,
+				     "no remotes available");
```

Comment:
it could still be a translated error though, right?

Reply:
It's not usual to translate the error domain. I think the best way to do this is provide a translation of the error code.